server/db/models/Appointment.js

```diff
@@ -0,0 +1,32 @@
+const mongoose = require("mongoose");
+
+const Schema = mongoose.Schema;
+
+const schema = new Schema({
+    meeting_id: {
+        type: mongoose.ObjectId,
```

Comment:
```suggestion
        type: mongoose.Schema.Types.ObjectId,
```
This is preferred when defining a Schema